extensions/have-a-cookie/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "have-a-cookie",
+  "title": "Have a Cookie",
+  "description": "Retrieves any cookie from Google Chrome.",
```

```suggestion
  "description": "Search and retrieve any cookies from Google Chrome",
```

I think I would do this, but again, not critically.